From 2d48a370cd225b13007bbbe6ef53ae2850014662 Mon Sep 17 00:00:00 2001
From: Irina Nita <irina.nita@posteo.com>
Date: Sun, 26 Apr 2026 01:33:33 +0300
Subject: [PATCH] modify fw image script

---
 build-firmware-image.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build-firmware-image.sh b/build-firmware-image.sh
index a86d9d9..af3016a 100755
--- a/build-firmware-image.sh
+++ b/build-firmware-image.sh
@@ -16,7 +16,7 @@ do_build()
 
     local fip_tool_opts=(
         --tb-fw "$PLATFORM_OUT_DIR/intermediates/tf-bl2.bin"
-        --nt-fw "$PLATFORM_OUT_DIR/intermediates/uefi.bin"
+        --nt-fw "$PLATFORM_OUT_DIR/intermediates/helloworld.bin"
         --soc-fw "$PLATFORM_OUT_DIR/intermediates/tf-bl31.bin"
         --fw-config "$PLATFORM_OUT_DIR/intermediates/morello_fw_config.dtb"
         --hw-config "$PLATFORM_OUT_DIR/intermediates/morello.dtb"
-- 
2.53.0

